vespamalloc/src/vespamalloc/malloc/allocchunk.h

```diff
@@ -20,16 +20,48 @@ struct TaggedPtr {
     TaggedPtr() noexcept : _ptr(nullptr), _tag(0) { }
     TaggedPtr(void *h, size_t t) noexcept : _ptr(h), _tag(t) {}
 
+#if defined(__x86_64__)
+#define VESPA_USE_ATOMIC_TAGGEDPTR
+    TaggedPtr load(std::memory_order = std::memory_order_seq_cst) {
+        return *this;
+    }
+    void store(TaggedPtr ptr) {
```

Comment:
This reintroduces the ABA problem since we are not guaranteed that the store is atomic.

Reply:
I also noticed that it was used in a uint test so I not look at the details. I will do that and see if it is necessary.